constants/index.js

```diff
@@ -45,6 +45,10 @@ export const COLORS = {
   'default': ['#CAB0FF', '#9D8DFF', '#6D8DFF', '#4457BC'],
   'aqua': ['#4EC9DA', '#65C6BE', '#7DD8FF', '#86DDB4'],
 };
+export const LOGOS = {
+  'default' : require('../../assets/logo.png'),
```

Comment:
I think it should just be one ../ for these routes

Reply:
No cuz this is the path relative to Home.jsx, not index.js

Reply:
I think Cole is right here, when I try to run the app as is it gives an error, but when I switch it to just one ../ it runs as expected

Reply:
yeah he was right forgot to come back and update it